docs/en/observability/monitor-k8s/monitor-k8s-explore-logs-and-metrics.asciidoc

```diff
@@ -0,0 +1,52 @@
+[discrete]
+[[monitor-kubernetes-explore]]
+== Part 3: Explore logs and metrics
+
+Use {kib} to view the metric and log data collected by {agent}.
+Refer to the following sections for more on viewing your data.
+
+* <<monitor-k8s-explore-metrics>>.
+* <<monitor-k8s-explore-logs>>.
+
+[discrete]
+[[monitor-k8s-explore-metrics]]
+=== View performance and health metrics
+
+To view the performance and health metrics collected by {agent}, open
+{kib} and go to **{observability} → Metrics**.
+
+On the **Inventory** page, you can switch between different views to see an
+overview of the containers and pods running on Kubernetes:
+
+[role="screenshot"]
+image::images/metrics-inventory.png[Inventory page that shows Kubernetes pods]
+
+On the **Metrics Explorer** page, you can group and analyze metrics for the
+resources that you are monitoring.
+
+[role="screenshot"]
+image::images/monitor-k8s-metrics-explorer.png[Metrics dashboard that shows CPU usage for Kubernetes pods]
+
+Notice how everywhere you go in {kib}, there is a search bar that allows you to,
+you know, search for things. It’s a great way to filter views and zoom into
+things when you're looking for that needle in a haystack.
+
+[discrete]
+[[monitor-k8s-explore-logs]]
+=== View container and audit logs
+
+To view container and audit logs, open {kib} and go to *{observability} → Logs Explorer*
+With **Logs Explorer**, you can quickly search and filter your log data, get information about the structure of log fields, and display your findings in a visualization.
```

Comment:
I haven't used the Logs explorer yet but as far as I understand, you should be able to filter by integration and datastream as well.

If mentioning audit_logs, maybe we should tell more about it. I believe that you would need to have the K8s integration installed and the audit_logs datastream enabled in the configs. maybe a screenshot here could help. 

if you think that is too much information at this level, i would remove a mention to audit logs

Reply:
I do mention filtering by K8s integration just below this [here](https://github.com/elastic/observability-docs/pull/3581/files/f480472d28f3e46c3803c1c03fea512a0e3cb30a#diff-747933bceeb073cd93fdbd9473fc1e2e7894c1eae9406bf1a90c40ef670c6cf4R44-R47). Is that what you mean?